airflow/models/dag.py

```diff
@@ -912,12 +912,7 @@ def set_dag_runs_state(
             query = query.filter(DagRun.execution_date >= start_date)
         if end_date:
             query = query.filter(DagRun.execution_date <= end_date)
-        drs = query.all()
-
-        dirty_ids = []
-        for dr in drs:
-            dr.state = state
-            dirty_ids.append(dr.dag_id)
```

Comment:
I wonder why we needed that dirty_ids list. Does not see to be used :).

Reply:
That was my main reason to do this change...

Reply:
Yeah that is strange